src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -283,11 +293,67 @@ protected virtual void SafeInternalStop()
 
         public bool IsInstalled => File.Exists(ExePath);
 
+        private static string ExtractVersionStringFromUrl(string url)
+        {
+            // We're loosely match the "version" part of the zip file name
```

Comment:
grammar nitpick: `We're loosely matching` or `We'll loosely match`? (unless I'm missing something)